client/src/common/UserAvatar.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import { Avatar } from "@material-ui/core";
+
+const UserAvatar = ({ user, className }) => {
+  const classes = useStyles();
+
+  return user.avatarUrl ? (
+    <Avatar className={`${classes.avatar} ${className}`} />
```

Comment:
if there is a user.avatarUrl, then Avatar should have attribute `src={user.avatarUrl}`

Reply:
done